common/src/main/java/com/wynntils/models/items/items/game/CraftedConsumableItem.java

```diff
@@ -30,8 +46,19 @@ public GearTier getGearTier() {
         return GearTier.CRAFTED;
     }
 
+    public boolean isHealing() {
+        return getEffects().stream()
+                .filter(e -> e.type().equals("Heal"))
```

Comment:
Qodana is right :)

Reply:
Thanks for noticing this. I just habitually ignore Qodana... :-/